extension/encoding/otlpencodingextension/metadata.yaml

```
@@ -0,0 +1,9 @@
+type: otlpencoding
```

Let's call the encoding extension this way? Similar to the storage extensions e.g. file_storage

```suggestion
type: otlp_encoding
```

---

Changed.

---

Sorry, I meant the component type name only. The go module should not have underscores